### PR TITLE
Fix could not bring up interface enp1s0 inside guest issue

### PR DESCRIPTION
**Pull request by chunfuwen**
ifdown and ifup not supported in new release
Fix them by alternative commands:ip link set up/down

Signed-off-by: chunfuwen <chwen@redhat.com>

### CODE REVIEW

**Comment by chunfuwen** [comment]
paste pieces of failure error message:
```
2018-06-03 05:40:33,795 test             L0268 INFO | cleaning libvirtd logs...
2018-06-03 05:40:33,802 stacktrace       L0041 ERROR| 
2018-06-03 05:40:33,802 stacktrace       L0044 ERROR| Reproduced traceback from: /var/tmp/libvirt-ci/runtest/avocado-vt/avocado/avocado/core/test.py:864
2018-06-03 05:40:33,802 stacktrace       L0047 ERROR| Traceback (most recent call last):
2018-06-03 05:40:33,802 stacktrace       L0047 ERROR|   File "/var/tmp/libvirt-ci/runtest/avocado-vt/avocado-vt/avocado_vt/test.py", line 297, in runTest
2018-06-03 05:40:33,802 stacktrace       L0047 ERROR|     raise self.__status  # pylint: disable=E0702
2018-06-03 05:40:33,803 stacktrace       L0047 ERROR|   File "/var/tmp/libvirt-ci/runtest/avocado-vt/avocado-vt/avocado_vt/test.py", line 251, in setUp
2018-06-03 05:40:33,803 stacktrace       L0047 ERROR|     self._runTest()
2018-06-03 05:40:33,803 stacktrace       L0047 ERROR|   File "/var/tmp/libvirt-ci/runtest/avocado-vt/avocado-vt/avocado_vt/test.py", line 423, in _runTest
2018-06-03 05:40:33,803 stacktrace       L0047 ERROR|     run_func(self, params, env)
2018-06-03 05:40:33,803 stacktrace       L0047 ERROR|   File "/var/lib/avocado/data/avocado-vt/test-providers.d/downloads/io-github-autotest-libvirt/libvirt/tests/src/virsh_cmd/domain/virsh_domif_setlink_getlink.py", line 253, in run
2018-06-03 05:40:33,803 stacktrace       L0047 ERROR|     test.fail(error_msg)
2018-06-03 05:40:33,803 stacktrace       L0047 ERROR|   File "/var/tmp/libvirt-ci/runtest/avocado-vt/avocado/avocado/core/test.py", line 1052, in fail
2018-06-03 05:40:33,803 stacktrace       L0047 ERROR|     raise exceptions.TestFail(message)
2018-06-03 05:40:33,803 stacktrace       L0047 ERROR| avocado.core.exceptions.TestFail: Could not bring up interface enp1s0 inside guest
2018-06-03 05:40:33,803 stacktrace       L0048 ERROR| 
2018-06-03 05:40:33,803 test             L0869 DEBUG| Local variables:
2018-06-03 05:40:33,839 test             L0872 DEBUG|  -> self <class 'avocado_vt.test.VirtTest'>: 1-type_specific.io-github-autotest-libvirt.virsh.domif_setlink_getlink.positive_test.interface_mac.no_config.domain_name.running_guest.domif_setlink.setlink_down
```

File: libvirt/tests/src/virsh_cmd/domain/virsh_domif_setlink_getlink.py [hunk 1]
```diff
@@ -235,11 +235,11 @@ def check_update_device(vm, if_name, session):
                 error_msg = "Link state isn't up in guest"
 
             # Ignore status of this one
-            cmd = 'ifdown %s' % guest_if_name
+            cmd = 'ip link set down %s' % guest_if_name
             pattern = "Device '%s' successfully disconnected" % guest_if_name
```
Comment (kylazhang) [inline libvirt/tests/src/virsh_cmd/domain/virsh_domif_setlink_getlink.py]: Also need to update the pattern as no output for ip link cmd....

Comment (kylazhang) [reply libvirt/tests/src/virsh_cmd/domain/virsh_domif_setlink_getlink.py]: May need "ip link show up" do decide it is up or down

Comment (chunfuwen) [reply libvirt/tests/src/virsh_cmd/domain/virsh_domif_setlink_getlink.py]: update pattern and use ip addr show dev to check state after updating

**Pull request merged**